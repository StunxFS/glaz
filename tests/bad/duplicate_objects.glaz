fn main() {
	let mut x = 10;
	{
		let mut x = 10;
	}
}
