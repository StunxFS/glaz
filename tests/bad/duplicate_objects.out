tests/bad/duplicate_objects.glaz:4:14: error: the name `x` is defined multiple times
     | 
   4 | 		let mut x = 10;
     | 		          ^
glaz error: could not compile `duplicate_objects`, aborting due to previous error.
